internal/cmd/operator_uninstall.go

```diff
@@ -16,42 +19,73 @@ func newOperatorUninstallCmd(cfg *action.Configuration) *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "uninstall <operator>",
 		Short: "Uninstall an operator and operands",
-		Long: `Uninstall removes the subscription, operator and optionally operands managed by the operator as well as 
-the relevant operatorgroup.
-
-Warning: this command permanently deletes objects from the cluster. Running uninstall concurrently with other operations
-could result in undefined behavior. 
-
-The uninstall command first checks to find the subscription associated with the operator. It then 
-lists all operands found throughout the cluster for the operator
-specified if one is found. Since the scope of an operator is restricted by
-its operator group, this search will include namespace-scoped operands from the
-operator group's target namespaces and all cluster-scoped operands. 
-
-The operand-deletion strategy is then considered if any operands are found on-cluster. One of cancel|ignore|delete. 
-By default, the strategy is "cancel", which means that if any operands are found when deleting the operator abort the 
-uninstall without deleting anything. 
-The "ignore" strategy keeps the operands on cluster and deletes the subscription and the operator.
-The "delete" strategy deletes the subscription, operands, and after they have finished finalizing, the operator itself.
-
-Setting --delete-operator-groups to true will delete the operatorgroup in the provided namespace if no other active 
-subscriptions are currently in that namespace, after removing the operator. The subscription and operatorgroup will be 
-removed even if the operator is not found.`,
+		Long: `Uninstall removes the subscription, operator and optionally operands managed
+by the operator as well as the relevant operatorgroup.
+
+Warning: this command permanently deletes objects from the cluster. Running
+uninstall concurrently with other operations could result in undefined behavior.
+
+The uninstall command first checks to find the subscription associated with the
+operator. It then lists all operands found throughout the cluster for the
+operator specified if one is found. Since the scope of an operator is restricted
+by its operator group, this search will include namespace-scoped operands from
+the operator group's target namespaces and all cluster-scoped operands.
+
+The operand-deletion strategy is then considered if any operands are found
+on-cluster. One of cancel|ignore|delete. By default, the strategy is "cancel",
```

Comment:
WDYT about "abort" over "cancel"? For some reason I find cancel less easy to associate with what's happening than abort.

Reply:
I _much_ prefer abort. However, there _might_ already be external use of `cancel` that would make such a change more difficult. I'll look into it.

Reply:
The only stakeholder I know of for this enum name is the OpenShift Console, but it appears as if that project is using just the ListOperands action, and _not_ the kubectl operator uninstall CLI or action.

Therefore, I think this change is fine.